src/metabase/analytics/stats.clj

```diff
@@ -574,11 +583,86 @@
           :deployment_model                 (deployment-model)
           :startup_time_millis              (-> stats :startup_time_millis)
           :has_activation_signals_completed (completed-activation-signals?)})]
-    (mapv
-     (fn [[k v]]
-       {"key"   (name k)
-        "value" v})
-     instance-attributes)))
+    (m->kv-vec instance-attributes)))
+
+(mu/defn- get-translation-count
+  :- [:map [:ok :int] [:not-found :int] [:invalid-format :int] [:total :int]]
+  "Get and clear the entity-id translation counter. This is meant to be called during the daily stats collection process."
+  []
+  (let [counter (setting/get-value-of-type :json :entity-id-translation-counter)]
+    (merge counter {:total (apply + (vals counter))})))
+
+(mu/defn- clear-translation-count!
+  "We want to reset the eid translation count on every stat ping, so we do it here."
+  []
+  (u/prog1 eid-translation/default-counter
+    (setting/set-value-of-type! :json :entity-id-translation-counter <>)))
+
+(defn- ->snowplow-metric-info
+  "Collects Snowplow metrics data that is not in the legacy stats format. Also clears entity id translation count."
+  []
+  (let [one-day-ago (t/minus (t/offset-date-time) (t/days 1))
+        total-translation-count (:total (get-translation-count))
+        _ (clear-translation-count!)]
+    {:models                  (t2/count :model/Card :type :model :archived false)
+     :new_embedded_dashboards (t2/count :model/Dashboard
+                                        :enable_embedding true
```

Comment:
We've since broken this setting up into multiple ones. This seems outdated. No?

Reply:
I think this is according to the [docs](https://www.notion.so/metabase/Migrate-Anonymous-Stats-ping-to-Snowplow-18f155e235d34ba09f45518a866336e2?pvs=4#1414ef173a184569a2278760650971eb) or what specifically are you referring to @albertoperdomo ?

Reply:
There is no setting enable-embedding anymore, we broke this up into three (static, interactive and SDK). See https://github.com/metabase/metabase/pull/48154

Reply:
I will update the doc.

Reply:
@albertoperdomo I see how it looks similar to the setting, but that is not the setting, it is the column on the dashboard table. It hasn't changed.